tests/network/primary_network/attach_primary/test_attach_network.py

```
@@ -0,0 +1,115 @@
+Test VM network attachment behavior - RestartRequired condition
+
+Preconditions:
+    - Cluster with Openshift Virtualization installed
+
+Jira: https://redhat.atlassian.net/browse/CNV-87822 # <skip-jira-utils-check>
+"""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, Generator
```

Ruff (0.15.14)

[warning] Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)